packages/microcosm-www-next/src/pages/index.js

```diff
@@ -57,9 +58,10 @@ export default class IndexPage extends React.Component {
   onIntersection = observed => {
     let entry = observed[0]
     let section = parseInt(entry.target.dataset.section)
+    let isIntersecting = entry.intersectionRatio >= 1.0
```

Comment:
Is the `1.0` here the same threshold as `observeOptions.threshold`?

Reply:
yes, I will save that in a var